bats/tests/k8s/up-downgrade-k8s.bats

```diff
@@ -70,10 +71,24 @@ verify_images() {
     verify_images
 }
 
+# Remove all the kubectl clients from the .kuberlr directory.
+# Then run `kubectl`, and it should pull in the `kubectl` for
+# the current k8s version in that directory.
+
+verify_kuberlr_for_version() {
+    k8sVersion=$1
+    rm -f "${HOME}"/.kuberlr/"${OS}"-"${ARCH_FOR_KUBERLR}"/kubectl*
+    run kubectl version
+    assert_output --regexp "Client Version.*GitVersion:.v$k8sVersion"
+    run ls -l "$HOME/.kuberlr/${OS}-${ARCH_FOR_KUBERLR}"
+    assert_output --partial "kubectl$k8sVersion"
+}
+
 @test 'upgrade kubernetes' {
     rdctl set --kubernetes.version "$RD_KUBERNETES_VERSION"
     wait_for_apiserver "$RD_KUBERNETES_VERSION"
     wait_for_container_engine
+    verify_kuberlr_for_version "$RD_KUBERNETES_VERSION"
```

Comment:
I would prefer to keep this line in a separate test. It really has nothing to do with "upgrade kubernetes".

And if it fails it provides a nicer way to identify the kind of failure.

Same thing for the similar call below.